mlir/test/python/integration/dialects/transform.py

```diff
@@ -99,26 +99,28 @@ def basic(target: any_op_t()):
 # CHECK-LABEL: TEST: test_apply_patterns
 @construct_and_print_in_module
 def test_apply_patterns(module_):
-    M, N, K = 3, 5, 3
+    b, M, N, K = 1, 3, 5, 3
```

Comment:
This is fine for now, but remember, we're do the same with brgemm soon, so we'll need to (re)move this eventually for good.